src/api/queries/getMetricQueries.js

```diff
@@ -0,0 +1,35 @@
+ApiClient_.prototype.fetchGetMetric = function (metric, slug, from, to, timeBoundSuffix) {
+  var timeBound = ['1d', '7d', '30d', '60d', '90d', '180d', '365d', '2y', '3y', '5y', '10y']
+  var timeBoundMetrics = [
```

Comment:
This array will not be needed when only the functions that need timeBound will have it as argument.